inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/project/KnowledgeBaseDetailsPanel.html

```diff
@@ -208,26 +215,27 @@ <h3 wicket:id="name" class="panel-title"></h3>
         </div>
       </div>
       <div class="form-group row">
-        <div class="col-sm-offset-2 col-sm-10">
-          <div class="checkbox">
-            <label wicket:for="enabled">
-              <input wicket:id="enabled" type="checkbox" /> <wicket:label key="enabled" />
-            </label>
-          </div>
-          <div class="checkbox">
-            <label wicket:for="supportConceptLinking">
-              <input wicket:id="supportConceptLinking" type="checkbox" />
-                <wicket:label key="kb.supportConceptLinking" />
-            </label>
-          </div>
+        <label class="col-sm-2 control-label">
```

Comment:
What is the purpose of the changes in this block?

Reply:
Moving the "supportsConceptLinking"  checkbox to the "Type" step.